kafka/utils.go

```diff
@@ -0,0 +1,25 @@
+package kafka
+
+import (
+	confluent "github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/twinj/uuid"
+)
+
+// according to this we need to generate a uuid to set as group.id:
+// https://github.com/edenhill/librdkafka/issues/1210
```

Comment:
it looks like any unique value will do. so importing a uuid library for this seems overkill.
can't we simply use the metrictank instance name, which we also know to be unique?
(this may also help with troubleshooting)

Reply:
yeah good idea